app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardGeneralStep/PlanWizardGeneralStep.js

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
 import { Field, reduxForm } from 'redux-form';
-import { required } from 'redux-form-validators';
+import { required, length } from 'redux-form-validators';
 import { Form } from 'patternfly-react';
 import PropTypes from 'prop-types';
 import { FormField } from '../../../../../common/forms/FormField';
 import { BootstrapSelect } from '../../../../../common/forms/BootstrapSelect';
+import {
+  MAX_LENGTH_NAMES,
+  MAX_LENGTH_DESCRIPTIONS,
+  nameHelp,
+  descriptionHelp,
+  nameRequired,
+  nameTooLong,
+  descriptionTooLong
+} from '../../../../../../../../common/constants'; // Oh my
```

Comment:
hey this looks familiar.. looks a little WET... if only it could be DRY'd....

Reply:
yeah, yeah :) i'll go get the hair dryer, good old Cmd+Shift+F

Reply:
Wait, DRY is an acronym but did someone actually come up with a WET backronym?  Weakly Extracted Text?

Reply:
😆 https://en.wikipedia.org/wiki/Don't_repeat_yourself is one of the good ones

Reply:
haha yes, I know. haha. I prefer this wiki for coding mantras though. 😏 http://wiki.c2.com/?DontRepeatYourself